near-sdk/src/store/tree_map/mod.rs

```diff
@@ -9,10 +9,10 @@ use crate::store::LookupMap;
 use crate::{env, IntoStorageKey};
 use borsh::{BorshDeserialize, BorshSerialize};
 pub use entry::Entry;
-pub use iter::{Iter, IterMut, Keys, Range, RangeMut, Values, ValuesMut};
+pub use iter::{Iter, IterMut, KeysRange, Range, RangeMut, Values, ValuesMut};
```

Comment:
Oh... interesting... seems like the `KeysRange` type was not exported previously. That was a mistake :D

Reply:
You want both `Keys` and `KeysRange` in here?

Reply:
oh, actually. Sorry for the false positive here. No, KeysRange is only used internally; it was correct not to be exposed.